src/filesystem/impls/filer/lib/HTMLRewriter.js

```diff
@@ -140,7 +135,7 @@ define(function (require, exports, module) {
                 return;
             }
 
-            if(jsEnabled || jsEnabledOverride) {
+            if(PreferencesManager.get("allowJavaScript") || jsEnabledOverride) {
```

Comment:
small optimization here:
store the result of `PreferencesManager.get("allowJavaScript")` in a variable (maybe `allowJavascript`) on line 131 and use that variable here instead.